launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -147,15 +148,24 @@ def _load_nodes_in_docker(
                 executable=executable_name
             )
 
-            self._container.exec_run(
+            log_generator = self._container.exec_run(
```

Comment:
Do we need to check the exit code here for success?

Reply:
Its always going to be `None` due to `stream=True`